src/app/pages/DashboardPage/AverageTransactionSize.tsx

```diff
@@ -4,12 +4,19 @@ import Card from '@mui/material/Card'
 import CardHeader from '@mui/material/CardHeader'
 import CardContent from '@mui/material/CardContent'
 import { LineChart } from '../../components/charts/LineChart'
-import { useGetConsensusStatsTxVolume } from '../../../oasis-indexer/api'
+import { Layer, useGetLayerStatsTxVolume } from '../../../oasis-indexer/api'
 
 export const AverageTransactionSize: FC = () => {
   const { t } = useTranslation()
-  // TODO: Replace with real stats when available
-  const dailyVolumeQuery = useGetConsensusStatsTxVolume()
+  const dailyVolumeQuery = useGetLayerStatsTxVolume(
+    Layer.consensus, // TODO: which layer do we want here?
+    {
+      // TODO: Do we want some parameters for the stats? Like these:
+      // limit: 2,
+      // offset: 2,
+      // bucket_size_seconds: 60,
```

Comment:
I think we are happy with current defaults: `limit: 100, bucket_size_seconds: 24*60*60`

Reply:
ok